frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -67,40 +67,40 @@ function BookInfo:show(file, book_props)
     table.insert(kv_pairs, { _("Folder:"), BD.dirpath(filemanagerutil.abbreviate(directory)), separator = true })
 
     -- book_props may be provided if caller already has them available
-    -- but it may lack 'pages', that we may get from sidecar file
+    -- but it may lack "pages", that we may get from sidecar file
     if not book_props or not book_props.pages then
         -- check there is actually a sidecar file before calling DocSettings:open()
         -- that would create an empty sidecar directory
         if DocSettings:hasSidecarFile(file) then
             local doc_settings = DocSettings:open(file)
             if doc_settings then
                 if not book_props then
-                    -- Files opened after 20170701 have a 'doc_props' setting with
-                    -- complete metadata and 'doc_pages' with accurate nb of pages
-                    book_props = doc_settings:readSetting('doc_props')
+                    -- Files opened after 20170701 have a "doc_props" setting with
+                    -- complete metadata and "doc_pages" with accurate nb of pages
```

Comment:
Although I admit on this one not having a period gives it a nice balance of exactly the same width.